src/styles/theme.scss

```diff
@@ -198,7 +229,18 @@ h5 {
 .button--alternate:active,
 .button--alternate:focus,
 .button--alternate:hover {
-  background-color: $purple-light;
+  background-color: $pink-light;
+}
+
+.button[href*="tel"] {
```

Comment:
This should be taken care of in the core - we should check. Maybe the other ca brands didn't have phone here.